examples/material_gallery/lib/demo/menu_demo.dart

```diff
@@ -94,7 +94,7 @@ class MenuDemoState extends State<MenuDemo> {
                   child: new Text('Context menu item one')
                 ),
                 new PopupMenuItem(
-                  disabled: true,
+                  isDisabled: true,
```

Comment:
let's stick with the boolean.

As far as isFoo vs foo, for widget constructor arguments, we don't generally do the is prefix. isFoo is usually reserved for getters that examine state.